### PR TITLE
Styled components

### PR DESCRIPTION
**Pull request by menor**

### CODE REVIEW

File: src/components/Alert/Alert.js [hunk 1]
```diff
@@ -1,15 +1,9 @@
-import React, { Component, PropTypes } from 'react'
+import React, {Component, PropTypes} from 'react'
```
Comment (heinburger) [inline src/components/Alert/Alert.js]: we might want to think about importing from prop-types for v2. We dont need to do it here, so I will make an issue

File: src/elements/Input/Input.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import styled, { injectGlobal } from 'styled-components'
+<<<<<<< HEAD
```
Comment (heinburger) [inline src/elements/Input/Input.js]: ?

Comment (menor) [inline src/elements/Input/Input.js]: I used the awesome atom editor new behaviour here.

File: src/components/Alert/Alert.js (hunk 1 above)
Comment (menor) [inline src/components/Alert/Alert.js]: 👍

**Pull request merged**